src/js/tabs/osd.js

```diff
@@ -1428,16 +1441,16 @@ OSD.GUI.preview = {
         overflows_line = FONT.constants.SIZES.LINE - ((position % FONT.constants.SIZES.LINE) + display_item.preview.length);
         if (overflows_line < 0) {
             position += overflows_line;
-        }
-        
+        }        
```

Comment:
Space.

Reply:
You'd think I'd know better having developed python for several years!